timemachine/testsystems/data/mobley_820789.sdf

```diff
@@ -0,0 +1,39 @@
+mobley_820789
```

Comment:
Clarification: is this from `freesolv.sdf`?

(If so, may be more convenient in a later PR if I modified `fetch_freesolv()` to return a dict `mol_name` -> `mol` instead of a list of `mol`s...)

Reply:
Yes, it could be useful for later.